Nexus.Sdk.Token/Responses/SignableResponse.cs

```diff
@@ -18,6 +18,9 @@ public record SignablePayoutResponse : SignableResponse
 {
     [JsonPropertyName("payout")]
     public required PayoutResponse PayoutOperationResponse { get; set; }
+
+    [JsonPropertyName("transactionEnvelope")]
+    public required TxEnvelopeResponse TransactionEnvelopeResponse { get; set; }
```

Comment:
This is already a part of the response. Please remove it.

Reply:
[Remove TxEnvelopeResponse because it exists already](https://github.com/QuantozTechnology/Nexus.Sdk.Dotnet/pull/37/commits/098e71cb6bda1238cda1156a9d3349bf30cf80e8)